__tests__/magic-block-parser.test.js

```diff
@@ -230,14 +228,11 @@ describe('Parse Magic Blocks', () => {
       "backgroundColor": "#ffffff",
       "title": "Tutorial Title",
       "emoji": "🦉",
-      "link": "http://test.com",
+      "link": "http://test.com"
```

Comment:
Having trailing commas here still ok?

Reply:
I don't think so. It's never OK for `JSON.parse`, right?

Reply:
yeah that'll bust json.parse